docs/native-node-modules.md

```diff
@@ -18,7 +18,7 @@ when releasing.
 Install the pre-requisites for your system:
 
 * [Windows pre-requisites](https://github.com/vector-im/element-desktop/blob/develop/docs/windows-requirements.md)
-* Linux: TODO
+* [Linux pre-requisites](https://github.com/ttheshreeyasingh/element-desktop/blob/develop/docs/linux-requirements.md)
```

Comment:
This is linking to your fork - it should be similar to the line above, linking to the vector-im repo.